frontend/src/pages/signup/components/SignupForm/SignupForm.tsx

```diff
@@ -65,8 +65,15 @@ function SignupForm() {
     handleDuplicateConfirmClick,
     shouldBlockSubmitByUserId,
     getFinalUserIdErrorMessage,
+    resetDuplicateCheck,
+    duplicateChecked,
   } = useUserIdDuplicateCheck({ userId, userIdErrorMessage });
 
+  const onUserIdChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    handleUserIdChange(e);
+    resetDuplicateCheck();
+  };
+
```

Comment:
p4: userId 가 바뀔때마다 reset 시켜주는건 불필요한 것 같은데 useUserIdDuplicateCheck 훅에서 duplicateChecked 상태도 가져와서 다음과 같이 작성하는건 어떤것같아?

```suggestion
  const onUserIdChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    handleUserIdChange(e);
    if(duplicateChecked){
       resetDuplicateCheck();
    }
  };

```

Reply:
이렇게 하면 reset을 필요할때만 할 수 있겠다!! 반영할게~~